tests/test_ops/test_ops.py

```diff
@@ -304,6 +305,7 @@ def test_instance_norm(backend,
             dynamic_axes=dynamic_axes,
             output_names=['output'],
             save_dir=save_dir)
+    torch.backends.cudnn.enabled = cudnn_enable
```

Comment:
This is not safe, the test could failed before this statement.
Try teardown with a fuxture. https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended